Developer documentation for access handler (#60)
common-fate/glide: docs/releasing.md

```

Where `<TAG>` is the RC tag, e.g. `v0.2.0-rc2`.

# Running Granted Approvals Locally

> The goals with Granted Approvals was to keep local development environments as similar to deployments as possible. 
> This makes spinning up a dev environment super simple.

Reviewer: ```suggestion
The goal with Granted Approvals was to keep local development environments as similar to deployments as possible. This makes spinning up a dev environment super simple.
```

> Start by exporting some AWS credentials, using whatever method you like.
> Here we just run `assume` using [Granted](https://granted.dev/).

Reviewer: ```suggestion
Start by exporting some AWS credentials, using whatever method you like. Here we just run `assume` using [Granted](https://granted.dev/).
```

```
assume rolename
```

Then initiate the dev deployment by running: 
```
mage deploy:dev
```
The command will ask some prompts for naming your dev deployment and which region to deploy some of the resources into.

> It will create a CDK changeset and ask to continue with the provisioning, input yes.

Reviewer: ```suggestion
It will create a CDK changeset and ask to continue with the provisioning, input _yes_.
```

> Once its completed successfully you will recieve the following success message in your console:

Reviewer: ```suggestion
Once it has completed successfully you will receive the following success message in your console:
```
